frontend-new/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
We should use the nextjs eslint: https://nextjs.org/docs/basic-features/eslint

Reply:
Added `next/core-web-vitals` config to extends